### PR TITLE
Mount debt test

### PR DESCRIPTION
**Pull request by Guntrisoft**

### CODE REVIEW

**Comment by Guntrisoft** [comment]
@Tr4pSt3R @darokel @asmega

**Comment by Tr4pSt3R** [comment]
:+1:

File: app/controllers/debt_test_controller.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class DebtTestController < EmbeddedToolsController
+  def parent_template
+    'layouts/engine_unconstrained'
+  end
+
+  protected
+    def category_id
+      ''
```
Comment (asmega) [inline app/controllers/debt_test_controller.rb]: do we know what this should be?

File: Gemfile [hunk 2]
```diff
@@ -46,6 +46,7 @@ gem 'budget_planner', '~> 4.0.1.689'
 gem 'car_cost_tool', '~> 1.0.4.110'
 gem 'debt_advice_locator', '~> 2.0'
 gem 'debt_free_day_calculator', '~> 2.1.0'
+gem 'debt_test', git: 'git@github.com:moneyadviceservice/debt_test.git'
```
Comment (asmega) [inline Gemfile]: hopefully we can fix the build and point to the gem

File: app/controllers/debt_test_controller.rb (hunk 1 above)
Comment (Tr4pSt3R) [inline app/controllers/debt_test_controller.rb]: I will refer that to mani.

File: Gemfile (hunk 2 above)
Comment (darokel) [inline Gemfile]: The build is green, let's point to the gem

**Pull request merged**